modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -107,3 +146,10 @@ You must create the bootstrap and control plane machines at this time. Because
 some pods are deployed on compute machines by default, also create at least two
 compute machine before you install the cluster.
 ====
+
+ifeval::["{context}" == "installing-ibm-power"]
```

Comment:
It looks like you need to unset the variables like how they were set. You need to unset both variables for both ifevals

Reply:
That was bad. Fixed.